File: Voronezh/Bars.GkhGji.Regions.Voronezh/Entities/GASU/GASU.cs
using Bars.B4.DataAccess;
using Bars.Gkh.Entities;
using Bars.Gkh.Enums;
using Bars.GkhGji.Entities;
using Bars.GkhGji.Regions.Voronezh.Enums;
using Bars.GkhGji.Regions.BaseChelyabinsk.Enums.SMEV;
using System;

namespace Bars.GkhGji.Regions.Voronezh.Entities
{
    public class GASU : BaseEntity
    {      
        /// <summary>
        /// ID сообщения в системе СМЭВ
        /// </summary>
        public virtual string MessageId { get; set; }      

        /// <summary>
        /// Текущее состояние запроса
        /// </summary>
        public virtual RequestState RequestState { get; set; }
       
        /// <summary>
        /// Ответ от сервера
        /// </summary>
        public virtual String Answer { get; set; }
         
        /// <summary>
        /// Инициатор запроса
        /// </summary>
        public virtual Inspector Inspector { get; set; }

        /// <summary>
        /// дата запроса
        /// </summary>
        public virtual DateTime RequestDate { get; set; }

        /// <summary>
        /// дата запроса
        /// </summary>
        public virtual DateTime DateFrom { get; set; }

        /// <summary>
        /// дата запроса
        /// </summary>
        public virtual DateTime DateTo { get; set; }

        /// <summary>
        /// Инициатор запроса
        /// </summary>
        public virtual GasuMessageType GasuMessageType { get; set; }
    }
}

File: Habarovsk/Bars.GkhGji.Regions.Habarovsk/libs/B4/model/smev/GASU.js
Ext.define('B4.model.smev.GASU', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'GASU'
    },
    fields: [
        { name: 'Id' },
        { name: 'RequestDate' },
        { name: 'Answer' },
        { name: 'DateFrom' },
        { name: 'DateTo' },
        { name: 'Inspector' },
        { name: 'RequestState', defaultValue: 0 },
        { name: 'MessageId' },
        { name: 'GasuMessageType', defaultValue: 10  }
       
    ]
});